src/main/common/components/payment-intention/payment-date.ts

```diff
@@ -78,7 +78,8 @@ export abstract class AbstractPaymentDatePage<Draft> {
     res.render(this.getView(), {
       heading: this.getHeading(),
       form: form,
-      notice: notice ? notice : undefined
+      notice: notice ? notice : undefined,
+      disposableIncome: res.locals.draft.document.courtDetermination ? res.locals.draft.document.courtDetermination.disposableIncome : undefined
```

Comment:
Why can't disposable income be type Number ( where undefined can be checked)? Type - res.locals.draft.document.courtDetermination seems incorrect to me?

Reply:
it's a ternary operation. if there is a courtDetermination it will show the disposableIncome if there is not then it will be undefined

Reply:
Sorry! Github view was bit muddled up and gave impression of type declaration to me before.